nixos/modules/services/web-apps/movim.nix

```diff
@@ -707,7 +815,13 @@ in
 
       phpfpm.pools.${pool} =
         let
-          socketOwner = if (cfg.nginx != null) then config.services.nginx.user else cfg.user;
+          socketOwner =
```

Comment:
Maybe we should move this up to the other, very similar condition to have them in one place.